lib/util/utimes.js

```diff
@@ -3,7 +3,7 @@
 const fs = require('../fs')
 const u = require('universalify').fromPromise
 
-async function utimesMillis (path, atime, mtime) {
+async function utimesMillis (path, atime, mtime, cb) {
```

Comment:
Why `cb` here?

Reply:
Removed in [`1ce5d34` (#1021)](https://github.com/jprichardson/node-fs-extra/pull/1021/commits/1ce5d348c426ea20c5006389477e163211898b22).